doc/howtos/realsense.rst

```diff
@@ -0,0 +1,34 @@
+Intel® RealSense™ cameras in IoT Reference OS Kit
+#################################################
+
+Intel® RealSense™ cameras are supported in IoT Reference OS Kit using
+``librealsense`` (https://github.com/IntelRealSense/librealsense)
+library.  By default `librealsense` is installed into the computer
+vision profile.
+
+Separation of graphical and console examples
+============================================
+
+IoT Reference OS Kit doesn't come with a graphical environment. For this
+reason only the console ``librealsense`` examples are available in
+computer vision development builds. To test that a camera works
+properly, plug it into a powered USB3 hub and say ``cpp-headless`` to
```

Comment:
Thanks @pohly ... it was a real misunderstanding! ... but it does like a joke now that I have the right context ;-)